client/src/utils/CastVoteRecordFiles.ts

```diff
@@ -103,6 +103,35 @@ export default class CastVoteRecordFiles {
     this.allCastVoteRecords = castVoteRecords
   }
 
+  public export() {
+    return JSON.stringify({
```

Comment:
Why not just do the stringify at this level and leave the others out?

Reply:
No good reason. Thanks for the push.